integrations-and-supported-tools/sacred/notebooks/Neptune_Sacred.ipynb

```diff
@@ -47,7 +47,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "! pip install -U neptune[sacred] sacred torch torchvision"
+    "%pip install -U neptune[sacred] sacred torch torchvision"
```

Comment:
Learned something new everyday (e.g. the difference between these two). Is this something we're going to adopt more widely?

Reply:
Yup. I'll update all `! pip` to `%pip` in notebooks as and when they are touched